test/images/agnhost/logs-generator/logs_generator.go

```diff
@@ -38,27 +39,35 @@ var (
 	}
 )
 
+// CmdLogsGenerator is used by agnhost Cobra.
+var CmdLogsGenerator = &cobra.Command{
+	Use:   "logs-generator",
+	Short: "Outputs lines of logs to stdout uniformly",
+	Long:  "Outputs <linesTotal> lines of logs to stdout uniformly for <duration>",
+	Args:  cobra.MaximumNArgs(2),
```

Comment:
Shouldnt this be 0 args; the variables are given via flags not arguments.

Reply:
Oups, you're right. Then that means that pretty much all commands should have this at 0.